### ISSUE TITLE
Add ability to auto-generate WDL wrappers for tools using the barclay library

### ISSUE THREAD

**Issue by droazen** [broadinstitute/gatk#2480]

**Comment by droazen** [comment]
Stretch goal for the 4.0 release this summer.

**Comment by vdauwera** [comment]
Would love to see this tackled in the near future if possible. We had a script that did this for the 3.x series, let me or @knoblett know if you're interested in seeing what we had set up for that.

**Comment by LeeTL1220** [comment]
Please use the template in the WDL GATK repo doc that was shared.  Or we can modify that template.  I'd like the document to match what is generated automatically.  The template in that document includes optimizations and is quite portable.

**Comment by droazen** [comment]
Blocked by need for input/output annotations in barclay.
